LEGO1/lego/legoomni/include/legoanimationmanager.h

```diff
@@ -1,9 +1,69 @@
 #ifndef LEGOANIMATIONMANAGER_H
 #define LEGOANIMATIONMANAGER_H
 
+#include "animstate.h"
 #include "decomp.h"
 #include "mxcore.h"
 
+struct Character {
+	char* m_name;
+	undefined m_unk0x4[0x10]; // 0x04
+	MxBool m_active;          // 0x14
+	undefined m_unk0x15;      // 0x15
+	undefined m_unk0x16;      // 0x16
+	undefined m_unk0x17;      // 0x17
+};
+
+struct Vehicle {
+	char* m_name;        // 0x00
+	undefined4 m_unk0x4; // 0x04
+};
+
+void C100d8ca8_Handler();
+void C100d8cd8_Handler();
+
+// VTABLE: LEGO1 0x100d8ca8
+// SIZE 0x10
+class C100d8ca8 : public MxCore {
```

Comment:
Unknown classes and structs are typically formatted as `Unk` and then the file offset; or if they're only used a member, `Unknown` and then their relative member offset.

Reply:
Omni-based global unknown classes should be named `LegoUnknown1000dead`